db/migrate/20191122000001_predefined_roles.rb

```diff
@@ -0,0 +1,27 @@
+class PredefinedRoles < ActiveRecord::Migration[5.1]
+    def change
```

Comment:
Did you intentionally decide to use change instead of up?
I thought that up is used in all migration in Zammad.